tests/test_kernel32/test_handle.py

```diff
@@ -120,3 +122,86 @@ def test_process_dies_before_timeout(self):
         self.assertEqual(
             WaitForSingleObject(hProcess, library.INFINITE),
             library.WAIT_OBJECT_0)
+
+
+class TestGetHandleInformation(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.GetHandleInformation`
+    """
+    def test_get_handle_info_stdin(self):
+        _, library = dist.load()
+        stdin_handle = GetStdHandle(library.STD_INPUT_HANDLE)
+        handle_flags = GetHandleInformation(stdin_handle)
+        inherit = handle_flags & library.HANDLE_FLAG_INHERIT
+        expected = (0, 1)
+        self.assertIn(inherit, expected)
+
+    def _expected_inheritance(self):
+        # Python >= 3.4 creates non-inheritable handles (PEP 0446)
+        return 1 if sys.version_info[:2] < (3, 4) else 0
+
+    def test_get_handle_info_file(self):
+        _, library = dist.load()
+        # can't use mkstemp: not inheritable on Python < 3.4
+        tempdir = tempfile.mkdtemp()
+        self.addCleanup(os.rmdir, tempdir)
+        filename = os.path.join(tempdir, "test_file")
+        with open(filename, "w") as test_file:
+            self.addCleanup(os.unlink, filename)
+            test_file.write("data")
+            file_handle = handle_from_file(test_file)
+            handle_flags = GetHandleInformation(file_handle)
+            inherit = handle_flags & library.HANDLE_FLAG_INHERIT
+        expected = self._expected_inheritance()
+        self.assertEqual(inherit, expected)
+
+    def test_get_handle_info_socket(self):
+        ffi, library = dist.load()
+        sock = socket.socket()
+        self.addCleanup(sock.close)
+        sock_handle = ffi.cast("void *", sock.fileno())
+        handle_flags = GetHandleInformation(sock_handle)
+        inherit = handle_flags & library.HANDLE_FLAG_INHERIT
+        expected = self._expected_inheritance()
+        self.assertEqual(inherit, expected)
+
+
+class TestSetHandleInformation(TestCase):
```

Comment:
Global comment for all tests in this test case.  Would you have objections to calling `GetHandleInformation` after each call to `SetHandleInformation` and testing the output?  

Technically speaking from a Windows perspective so long as the call to `SetHandleInformation` succeeds (since we're checking the error code) it should be fine.  But if it were me, I'd add the extra test just to be sure because it ensures that we can modify and then read data from the same handle with two different API calls.